The Next Web Clone

Davidosky007 opened the pull request:
## The Next Web Clone: Responsive Design Part 1

In this project we:

* we made use of media queries to make the web page responsive
* we made use of the proper units 
* we made two breakpoints one for laptops and the other for tablets

henatan99 committed:
Adds header and front sections and responsvie styling

- Adds header section and style with reponsive feature
- Adds the front section with responsive feature

Davidosky007 committed:
modify navbar section

henatan99 committed:
Add 4 more sections, style and responsiveness

- Add latest-news, latest-funding, and business sections
- Add styling for the sections, and uses float extensively to take
  advantage for the responsiveness
- Do responsiveness for two break points - 768px and 1024px

henatan99 committed:
Modified general styling for max-width

Modifies general styling for max-width to adapt for different screen
sizes using media queries

Davidosky007 committed:
update README file and media query for navbar mobile view

henatan99 committed:
Adds latest-deals and footer sections

- Adds latest deals sections, add styles and responsiveness
- As usual float lay out is used for articles in this section so that it
  will be easier for the responsive styling
- Fooeter section is added with icons, footer nav links, and copyright
  info

Davidosky007 committed:
modify cover section and updated its style sheet

henatan99 committed:
More styles apply for the different section elements

 - More styles applied to make the page look like the original page
 - More responsive styles are applied as well

henatan99 committed:
Merge branch 'feature-2' of https://github.com/henatan99/the-next-web-responsive-page-clone into feature-2

henatan99 committed:
Resolve file conflict

henatan99 committed:
Adds more styling and fix some faults on footer section

Davidosky007 committed:
fix linter errors and modify all style sheets

Davidosky007 committed:
fix typo in the index.html file

Davidosky007 committed:
update index.html file

Davidosky007 committed:
add project screen shoot and fix style sheets

Davidosky007 committed:
fix latest-funding.css

Davidosky007 committed:
modify business.css

Davidosky007 committed:
modify latest-funding.css

jubaan reviewed (changes requested):
# Status: Changes Required :recycle: 

## :wave: :wave: Hello @Davidosky007 and @henatan99  . Thanks for submitting you project!

Things to notice:
:100: Descriptive README
:100: Descriptive PR message
:100: Awesome solid start!

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [ ] Images goes outside boundaries of website
![image](https://user-images.githubusercontent.com/25774518/95882038-d5bd0480-0d3e-11eb-8c98-08ba6cf5636d.png)

- [ ] Elements are not aligned in bigger screen desktops
![image](https://user-images.githubusercontent.com/25774518/95882879-d5713900-0d3f-11eb-930b-add17561630e.png)

- [ ] Footer doesn't match original
![image](https://user-images.githubusercontent.com/25774518/95883003-fb96d900-0d3f-11eb-8481-55604ff0a227.png)

- [ ] Navbar has issues in tablet view
![image](https://user-images.githubusercontent.com/25774518/95883147-3436b280-0d40-11eb-8afd-1bcd56dc3475.png)

- [ ] Latest funding doesn't match original in tablet view
![image](https://user-images.githubusercontent.com/25774518/95883226-4d3f6380-0d40-11eb-9344-b02896c412f8.png)

- [ ] Elements are not aligned in tablet view
![image](https://user-images.githubusercontent.com/25774518/95883332-6b0cc880-0d40-11eb-94dc-70eea1341afe.png)

- [ ] Footer doesn't look as expected in tablet view
![image](https://user-images.githubusercontent.com/25774518/95883418-84ae1000-0d40-11eb-8dfa-7193514ac212.png)

- [ ] Missing arrows for each section

- [ ] element doesn't look as expected in mobile view
![image](https://user-images.githubusercontent.com/25774518/95883561-b45d1800-0d40-11eb-8b3a-6920b399ddd3.png)

### [Comments] :green_circle: 

- A great tool to see if the website looks as expected in the browser media query inspector. Use it to review the original against the clone.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

Davidosky007 committed:
fix business.css, footer section, front.css, latest-deals.css and latest-funding.css

henatan99 committed:
Make changes based on TSEs code review comments
- Out of viweport content fixed
- Non-alignment of sections fixed
- Missing arrows fixed
- Footer section fixed to make it look like original page
- Nav bar items overlp fixed

henatan99 committed:
More fixes made ad=nd file conflicts resolved

henatan99 committed:
Resolve file conflicts and make few fixes

henatan99 committed:
Fix lynting errors

henatan99 committed:
modifies read me

jubaan reviewed (approved):
# :tada: :tada: :tada: :rotating_light: Status: Approved :rotating_light:  :tada: :tada: :tada:

#### :clap: :clap: Great job :hammer_and_pick:!! There are no more issues left to be addressed. 

## You may now merge to master! :+1: :100:

## See you at the top :rocket:!

# :tada: :tada: :tada: :rotating_light: Status: Approved :rotating_light:  :tada: :tada: :tada:
